<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>updateEmployeewindow</class>
 <widget class="QDialog" name="updateEmployeewindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>541</width>
     <height>651</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(205, 205, 205); 
border-radius: 12px;
 </string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>0</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color:rgb(205, 205, 205);
    color: white;
     border-top-left-radius: 0px;    
    border-top-right-radius: 12px;   
    border-bottom-left-radius: 0px;   
    border-bottom-right-radius: 0px;  
    font-size: 20px;
}
QPushButton:hover {
    background-color:rgb(133, 133, 133);
}</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>481</width>
      <height>601</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>190</y>
       <width>421</width>
       <height>391</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2,1,2">
        <item>
         <widget class="QLabel" name="EmployeeIDlabel">
          <property name="font">
           <font>
            <family>等线</family>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#5a5a5a;&quot;&gt;· 职工编号：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="EmployeeIDlineEdit">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="toolTip">
           <string/>
          </property>
          <property name="styleSheet">
           <string notr="true">border:1px solid rgb(90, 90, 90);
border-radius:4px;
</string>
          </property>
          <property name="placeholderText">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="Namelabel">
          <property name="font">
           <font>
            <family>等线</family>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#5a5a5a;&quot;&gt;· 姓名：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="NamelineEdit">
          <property name="styleSheet">
           <string notr="true">border:1px solid rgb(90, 90, 90);
border-radius:4px;
</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2,1,2">
        <item>
         <widget class="QLabel" name="BaseSalarylabel">
          <property name="font">
           <font>
            <family>等线</family>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#5a5a5a;&quot;&gt;· 基本工资：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="BaseSalarylineEdit">
          <property name="styleSheet">
           <string notr="true">border:1px solid rgb(90, 90, 90);
border-radius:4px;
</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="Allowanceslabel">
          <property name="font">
           <font>
            <family>等线</family>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#5a5a5a;&quot;&gt;· 津贴：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="AllowanceslineEdit">
          <property name="styleSheet">
           <string notr="true">border:1px solid rgb(90, 90, 90);
border-radius:4px;
</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,2,1,2">
        <item>
         <widget class="QLabel" name="PositionAllowancelabel">
          <property name="font">
           <font>
            <family>等线</family>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#5a5a5a;&quot;&gt;· 岗贴：  &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="PositionAllowancelineEdit">
          <property name="styleSheet">
           <string notr="true">border:1px solid rgb(90, 90, 90);
border-radius:4px;
</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="Subsidieslabel">
          <property name="font">
           <font>
            <family>等线</family>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#858585;&quot;&gt;·&lt;/span&gt;&lt;span style=&quot; font-size:12pt; color:#5a5a5a;&quot;&gt; 补贴：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="SubsidieslineEdit">
          <property name="styleSheet">
           <string notr="true">border:1px solid rgb(90, 90, 90);
border-radius:4px;
</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,2,1,2">
        <item>
         <widget class="QLabel" name="HousingAllowancelabel">
          <property name="font">
           <font>
            <family>等线</family>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#5a5a5a;&quot;&gt;· 房贴：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="HousingAllowancelineEdit">
          <property name="styleSheet">
           <string notr="true">border:1px solid rgb(90, 90, 90);
border-radius:4px;
</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="TransportationAllowancelabel">
          <property name="font">
           <font>
            <family>等线</family>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#5a5a5a;&quot;&gt;· 交通补贴：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="TransportationAllowancelineEdit">
          <property name="styleSheet">
           <string notr="true">border:1px solid rgb(90, 90, 90);
border-radius:4px;
</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,2,1,2">
        <item>
         <widget class="QLabel" name="Rentlabel">
          <property name="font">
           <font>
            <family>等线</family>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#5a5a5a;&quot;&gt;· 房租：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="RentlineEdit">
          <property name="styleSheet">
           <string notr="true">border:1px solid rgb(90, 90, 90);
border-radius:4px;
</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="MembershipFeeslabel">
          <property name="font">
           <font>
            <family>等线</family>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#5a5a5a;&quot;&gt;· 会费：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="MembershipFeeslineEdit">
          <property name="styleSheet">
           <string notr="true">border:1px solid rgb(90, 90, 90);
border-radius:4px;
</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,2,3">
        <item>
         <widget class="QLabel" name="Savingslabel">
          <property name="font">
           <font>
            <family>等线</family>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#5a5a5a;&quot;&gt;· 储蓄：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="SavinglineEdit">
          <property name="styleSheet">
           <string notr="true">border:1px solid rgb(90, 90, 90);
border-radius:4px;
</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="confirmButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>等线</family>
            <pointsize>-1</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
height:34;
width:151;
color:rgb(90, 90, 90);
border:1px solid rgb(90, 90, 90);
border-radius:4px;
font-size:10px;
}
QPushButton:hover {
    background-color: rgb(205, 205, 205);
color:rgb(255, 255, 255);
font-size:12px;
}
QPushButton:pressed{
    background-color: rgb(133,133,133);
color:rgb(255, 255, 255);
font-size:10px;
}</string>
          </property>
          <property name="text">
           <string>确定</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="imageLabel">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>131</width>
       <height>131</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>icon/default.png</pixmap>
     </property>
    </widget>
    <widget class="QPushButton" name="selectImageBtn">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>70</y>
       <width>153</width>
       <height>36</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>等线</family>
       <pointsize>-1</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
height:34;
width:151;
color:rgb(90, 90, 90);
border:1px solid rgb(90, 90, 90);
border-radius:4px;
font-size:10px;
}
QPushButton:hover {
    background-color: rgb(205, 205, 205);
color:rgb(255, 255, 255);
font-size:12px;
}
QPushButton:pressed{
    background-color: rgb(133,133,133);
color:rgb(255, 255, 255);
font-size:10px;
}</string>
     </property>
     <property name="text">
      <string>选择头像</string>
     </property>
    </widget>
   </widget>
   <zorder>frame_2</zorder>
   <zorder>closeButton</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
